### PR TITLE
Edit and delete

### PR DESCRIPTION
**Pull request by TarandeepNandhra**
Added Delete and Update route for workouts and updated README

### CODE REVIEW

File: test/server.test.js [hunk 1]
```diff
@@ -158,3 +158,97 @@ test("GET /workout:id failing request", async () => {
...
+    image: "buffarms.png"
+   }]
+
+   const workoutUpdated = {
+     workoutTitle,
+     exercises
+   }
+
+  await supertest(app)
+		.put(`/workout/${workout._id}`)
```
Comment (TataSher) [inline test/server.test.js]: Something weird with the indentation

File: test/server.test.js [hunk 2]
```diff
@@ -158,3 +158,97 @@ test("GET /workout:id failing request", async () => {
...
+    .send(workoutUpdated)
```
Comment (TataSher) [inline test/server.test.js:227]: Weird indentation

**Review by TataSher — COMMENTED** [review]
Looks Great!

File: README.md [hunk 3]
```diff
@@ -16,3 +20,53 @@ Dev:
 - Nodemon: auto restarts server
 - Jest
 - Supertest
+
+## API routes
+
+| Endpoint          | Functionality            | Deployed         |
```
Comment (TataSher) [inline README.md:26]: Awesome that you added the table and examples! Super clear

File: test/server.test.js (hunk 1 above)
Comment (ralphbartley) [reply test/server.test.js]: yep!

File: test/server.test.js (hunk 2 above)
Comment (ralphbartley) [reply test/server.test.js:227]: indeed!

File: README.md (hunk 3 above)
Comment (ralphbartley) [reply README.md:26]: Yeah it's cool isn't it?  Handy table Taran pulled up

**Pull request merged**